README.md

```diff
@@ -30,91 +30,127 @@ ParseDate has class methods for date string parsing.
 ```
 require 'parse_date'
 
-ParseDate.earliest_year('12/25/00') # 2000
```

Comment:
the only change i made to the existing lines was to put the expected values tabbed to the right